lib/moon/design/popover.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Moon.Design.Popover do
+  @moduledoc false
+
+  use Moon.StatefulComponent
+
+  prop(on_click, :event)
+  prop(close, :event)
```

Comment:
any reasons for exporting this event outside? do we need it to be customizable?

Reply:
But these props need to be there because of lines 19-20?
 :on-click={@on_click || "toggle_open"}
      :on-click-away={@close || "close_panel"}